internal/static/cartogram2.js

```diff
@@ -450,12 +452,28 @@ class MapVersion {
         this.extrema = extrema;
         this.labels = labels;
         this.world = world;
+        // legendData stores legend and gridline information of the map version.
+        this.legendData = {
+            "gridData": {
+                "gridA": {width: null, scaleNiceNumber: null, gridPath: null},
+                "gridB": {width: null, scaleNiceNumber: null, gridPath: null},
+                "gridC": {width: null, scaleNiceNumber: null, gridPath: null}
```

Comment:
We only use these three fixed grid versions throughout the CartMap class; I thought if we have them specified, it will be easier for future developers to understand the code more easily. Also, I used an object instead of an array, because if I use an array, I would need to access the items via indexing which would be unclear to new developers as they would need to repetitively refer back to the definition to keep track of what each index number refers to.